### DISCUSSION TITLE
Badge Concours

### DISCUSSION THREAD

**Discussion by Jaarzz** [Crownicles/Crownicles#1132] [Suggestions fermées]
Je propose d'organiser de temps en temps des concours avec à la clé un badge personnalisé. Voilà

**Comment by BastLast** [comment]
Hello, le nombre de badge que nous pouvons créer est limité à environ une vingtaine, il n'y aura donc probablement jamais de badges personnalisés.

**Reply by QQtin3** [reply]
> 

Hm hm, pas vraiment pour la limite..

**Reply by BastLast** [reply]
Ya pas de hard limite technique mais une somme de petites contraintes relou liées a l'augmentation du nombre de badge qui font que j'aimerai limiter leur nombre x)

**Reply by Pietagorh** [reply]
Beh plus maintenant ?
